packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -0,0 +1,479 @@
+import * as React from "react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { get } from "lodash-es";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+import { SelectClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { SelectHandledProps, SelectProps, SelectUnhandledProps } from "./select.props";
+import { ListboxItemProps } from "../listbox-item";
+import Listbox from "../listbox";
+import Button from "../button";
+
+export interface SelectState {
+    value: string | string[];
+    displayString: string;
+    selectedItems: ListboxItemProps[];
+    isMenuOpen: boolean;
+}
+
+class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, SelectState> {
+    public static displayName: string = "Select";
+
+    public static defaultProps: Partial<SelectProps> = {
+        multiselectable: false,
+        disabled: false,
+        defaultSelection: [],
+        placeholder: "",
+    };
+
+    private static idPropertyKey: string = "id";
+
+    /**
+     * Handled props instantiation
+     */
+    protected handledProps: HandledProps<SelectHandledProps> = {
+        isMenuOpen: void 0,
+        disabled: void 0,
+        displayStringFormatterFunction: void 0,
```

Comment:
"Function" seems redundant, it's like calling something myIndexVariable.

I would prefer:

displayStringFormatterFunction => displayStringFormatter
contentDisplayRenderFunction => contentFormatter (is this what it is doing?)
menuRenderFunction => menuFormatter

Reply:
Did this:
displayStringFormatterFunction => displayStringFormatter
contentDisplayRenderFunction =>trigger
menuRenderFunction => no longer exists